src/Algebra/Bundles.agda

```diff
@@ -945,12 +945,6 @@ record Quasigroup c ℓ : Set (suc (c ⊔ ℓ)) where
   open RawQuasigroup rawQuasigroup public
     using (_≈_; //-rawMagma; \\-rawMagma; ∙-rawMagma)
 
-  setoid : Setoid _ _
-  setoid = record { isEquivalence = isEquivalence }
-
-  open Setoid setoid public
-    using (_≉_)
```

Comment:
Where does this now get re-exported from? Looks to me as if it's just gone?

Reply:
This was a mistake so I removed it. It should be in structure and not bundle